db/liquibase/changesets/20180915_collection.xml

```diff
@@ -24,8 +24,8 @@
                 <constraints nullable="false"/>
             </column>
         </createTable>
-        <createIndex tableName="collection" indexName="IDX_collection_name">
-            <column name="name"/>
+        <createIndex tableName="collection" indexName="IDX_collection_external_id">
```

Comment:
Postgres creates an index automatically for unique constraints. I'm guessing that saying `unique="true"` on the column will do that but I haven't checked. We may find that we won't need this index, but that is easy enough to drop (and shouldn't have negative impacts on table storage... but who knows). Anyway, again, good to merge as-is.